wf/ja-jp/Content/Topics/Workflows/workflow-connect-your-applications.htm

```diff
@@ -297,23 +297,23 @@ <h1>アプリケーションの接続</h1>
                                                             </li>
                                                         </ul>
                                                         <h2><a name="Authenti"></a>Oktaアカウントの認証</h2>
-                                                        <p>Workflowsコンソールで何か作業を始める前に、<span class="mc-variable okta-variables.ProductName variable">Okta</span>アカウントを認証してください。認証は一度だけ必要です。認証を行う際には、<span class="mc-variable okta-feature-names.Administrator_dashboard variable">管理コンソール</span>からWorkflowsコンソールに情報をコピーする必要があるため、<span class="mc-variable okta-feature-names.Administrator_dashboard variable">管理コンソール</span>は開いたままにしておいてください。 </p>
+                                                        <p>Workflowsコンソールで何か作業を始める前に、<span class="mc-variable okta-variables.ProductName variable">Okta</span>アカウントを認証してください。認証は一度だけ必要です。認証を行う際には、<span class="mc-variable okta-feature-names.Administrator_dashboard variable">Admin Console</span>からWorkflowsコンソールに情報をコピーする必要があるため、<span class="mc-variable okta-feature-names.Administrator_dashboard variable">Admin Console</span>は開いたままにしておいてください。 </p>
```

Comment:
@pengyao-okta I brought it up elsewhere, marking it here for WF: Are we now leaving `Admin Console` untranslated? Seems odd to me, but maybe there's reasons.

Reply:
Hi @paulwallace-okta , yes, it is DNT, should not be translated.